plugins/mcreator-localization/lang/texts.properties

```diff
@@ -1892,7 +1892,7 @@ elementgui.armor.leggings_need_name=Armor leggings need a name
 elementgui.armor.helmet_needs_name=Armor helmet needs a name
 elementgui.armor.armor_needs_texture=Armor needs to have a texture
 elementgui.armor.helmet={0} Helmet
-elementgui.armor.body={0} Body
+elementgui.armor.body={0} Chestplate
```

Comment:
You could change this localization key to `elementgui.armor.chestplate`, as well as update `elementgui.armor.body_needs_name` and `elementgui.armor.body_name` in a similar way

Reply:
I changed all references to Body to Chestplate in the texts. The only place I kept it is inside the dropdown menu (of custom Java models) as I'm not sure if it's safe or not for older workspaces to change it (without adding a converter)